static/i18n/en/app.json

```diff
@@ -250,23 +299,6 @@
       "title": "Choose a provider to swap crypto",
       "learnMore": "What is Swap?",
       "kycRequired": "KYC Required",
-      "wyre": {
-        "bullets": [
-          "US-only (soon Europe, Africa and Asia)",
-          "Multi coins and tokens",
-          "Centralized"
-        ]
-      },
-      "changelly": {
-        "bullets": ["BTC, ETH, USDT and 50+ others", "Multi-currency", "Centralized"]
-      },
-      "paraswap": {
-        "bullets": [
-          "Connect to 30+ DEXs (Uniswap, SushiSwap,...)",
-          "Ethereum & ERC20 tokens",
-          "Decentralized & Trustless"
-        ]
-      },
```

Comment:
can you clarify the removing of these translations?
where they simply unused?

Reply:
There were used on the provider selector screen, which doesn't exist anymore.

Reply:
In fact, regular runs of `python3 scripts/i18next-unused.py` can reveal whole literal namespaces that are zombie 🧟